.github/workflows/pr-akka-samples.yml

```diff
@@ -0,0 +1,94 @@
+name: Update and PR akka-samples
+
+on:
+  workflow_dispatch: # to trigger manually
+  workflow_run:
+    workflows: ["Publish"]
+    types:
+      - completed
+
+jobs:
+  debug:
+    if: github.event_name != 'workflow_dispatch'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Debugging
+        run: |
+          echo "Conclusion from `Publish` workflow: ${{ github.event.workflow_run.conclusion }}
+          echo "Tag created from `Publish` workflow: ${{ github.ref }}
+
+  samples-shopping-cart-quickstart:
```

Comment:
Would be nice if it could auto-enumerate samples so we don't have to change this job when the set of samples changes. Maybe too tricky to be worth it with all the steps?

Reply:
and we will have samples that are only in this repo and not mirrored in akka-samples. maybe we could use a matrix, to make it easy to add more